p2panda-sync/src/log_sync.rs

```diff
@@ -38,6 +38,7 @@ where
     }
 }
 
+// @TODO(adz): Does this need a version number?
```

Comment:
Hmm...good question. I'm currently leaning towards no. I suppose the other option for versioning is to include it in the sync protocol handshake (if necessary). Feels fine to leave out for now.

Reply:
Yeh, good question! We don't actually do anything with this yet, if we want to check versions on sync then we would need to add a message to send over the version during the handshake. We should definitely do this once things stabilise. For now I'm happy with it in or out.